<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Replays</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLineEdit" name="replayLine_1"/>
            </item>
            <item>
             <widget class="QPushButton" name="replayBrowse_1">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLineEdit" name="replayLine_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="replayBrowse_2">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLineEdit" name="replayLine_3"/>
            </item>
            <item>
             <widget class="QPushButton" name="replayBrowse_3">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLineEdit" name="replayLine_4"/>
            </item>
            <item>
             <widget class="QPushButton" name="replayBrowse_4">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLineEdit" name="replayLine_5"/>
            </item>
            <item>
             <widget class="QPushButton" name="replayBrowse_5">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Maps</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16">
            <item>
             <widget class="QLineEdit" name="mapLine_1"/>
            </item>
            <item>
             <widget class="QPushButton" name="mapBrowse_1">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLineEdit" name="mapLine_2"/>
            </item>
            <item>
             <widget class="QPushButton" name="mapBrowse_2">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLineEdit" name="mapLine_3"/>
            </item>
            <item>
             <widget class="QPushButton" name="mapBrowse_3">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="mapLine_4"/>
            </item>
            <item>
             <widget class="QPushButton" name="mapBrowse_4">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLineEdit" name="mapLine_5"/>
            </item>
            <item>
             <widget class="QPushButton" name="mapBrowse_5">
              <property name="text">
               <string>Browse...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QProgressBar" name="progressBar">
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="loadButton">
            <property name="text">
             <string>Load</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Made by Repo
osu.ppy.sh/users/2365927 | github.com/rgbeing
Suggestions &amp; bug reports are welcomed via osu! PM or Discord: repo2x</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
